resources/multisite_drupal_standard.make

```diff
@@ -739,7 +739,8 @@ projects[tmgmt][patch][] = https://www.drupal.org/files/issues/2812863.patch
 ; #2362321 : Check source length limits
 ; https://www.drupal.org/node/2362321
 ; https://webgate.ec.europa.eu/CITnet/jira/browse/NEPT-1802
```

Comment:
Hi,
Maybe you can also remove this line except if the new patch covers both issues.